<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab_1">
      <attribute name="title">
       <string>文件外部排序</string>
      </attribute>
      <widget class="QPushButton" name="pushButton_2">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>100</y>
         <width>80</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>外部排序</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>170</y>
         <width>80</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>创建内存索引</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>K线图</string>
      </attribute>
      <widget class="QComboBox" name="cb2">
       <property name="geometry">
        <rect>
         <x>116</x>
         <y>89</y>
         <width>188</width>
         <height>23</height>
        </rect>
       </property>
       <property name="editable">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="ConsoleStockCodeLabel">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>60</y>
         <width>60</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>股票代码：</string>
       </property>
      </widget>
      <widget class="QComboBox" name="cb1">
       <property name="geometry">
        <rect>
         <x>116</x>
         <y>60</y>
         <width>188</width>
         <height>23</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="editable">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="ConsoleMonthLabel">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>89</y>
         <width>60</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>交易年月：</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_5">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>140</y>
         <width>80</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>确定</string>
       </property>
      </widget>
      <widget class="QCustomPlot" name="custom_plot" native="true">
       <property name="geometry">
        <rect>
         <x>380</x>
         <y>180</y>
         <width>500</width>
         <height>500</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>热力系数图</string>
      </attribute>
      <widget class="QComboBox" name="cbb1">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>170</y>
         <width>131</width>
         <height>23</height>
        </rect>
       </property>
      </widget>
      <widget class="QComboBox" name="cbb3">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>210</y>
         <width>131</width>
         <height>23</height>
        </rect>
       </property>
      </widget>
      <widget class="QComboBox" name="cbb2">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>170</y>
         <width>131</width>
         <height>23</height>
        </rect>
       </property>
      </widget>
      <widget class="QComboBox" name="cbb4">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>210</y>
         <width>131</width>
         <height>23</height>
        </rect>
       </property>
      </widget>
      <widget class="QComboBox" name="cbb5">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>260</y>
         <width>131</width>
         <height>23</height>
        </rect>
       </property>
      </widget>
      <widget class="QComboBox" name="cbb7">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>300</y>
         <width>131</width>
         <height>23</height>
        </rect>
       </property>
      </widget>
      <widget class="QComboBox" name="cbb6">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>260</y>
         <width>131</width>
         <height>23</height>
        </rect>
       </property>
      </widget>
      <widget class="QComboBox" name="cbb8">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>300</y>
         <width>131</width>
         <height>23</height>
        </rect>
       </property>
      </widget>
      <widget class="QComboBox" name="cbb9">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>340</y>
         <width>131</width>
         <height>23</height>
        </rect>
       </property>
      </widget>
      <widget class="QComboBox" name="cbb10">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>340</y>
         <width>131</width>
         <height>23</height>
        </rect>
       </property>
      </widget>
      <widget class="QComboBox" name="cbb01">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>490</y>
         <width>131</width>
         <height>23</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>460</y>
         <width>131</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>请选择年月</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>140</y>
         <width>131</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>请选择十只股票</string>
       </property>
      </widget>
      <widget class="QCustomPlot" name="custom_plot_2" native="true">
       <property name="geometry">
        <rect>
         <x>360</x>
         <y>80</y>
         <width>600</width>
         <height>600</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_3">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>390</y>
         <width>80</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>确认</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_4">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>540</y>
         <width>80</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>绘制</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>600</y>
         <width>261</width>
         <height>161</height>
        </rect>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;选择好股票代码后点击确认&lt;/p&gt;&lt;p&gt;时间列表会自动筛选出十只股票共存的年月&lt;/p&gt;&lt;p&gt;选择后点击绘制。-1.01表示两只股票的收益&lt;/p&gt;&lt;p&gt;序列长度不一致 无法进行计算&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_4">
      <attribute name="title">
       <string>价格变动预测</string>
      </attribute>
      <widget class="QCustomPlot" name="custom_plot_3" native="true">
       <property name="geometry">
        <rect>
         <x>310</x>
         <y>150</y>
         <width>800</width>
         <height>600</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="ConsoleMonthLabel_2">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>129</y>
         <width>60</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>交易年月：</string>
       </property>
      </widget>
      <widget class="QComboBox" name="cb2_2">
       <property name="geometry">
        <rect>
         <x>106</x>
         <y>129</y>
         <width>188</width>
         <height>23</height>
        </rect>
       </property>
       <property name="editable">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="ConsoleStockCodeLabel_2">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>100</y>
         <width>60</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>股票代码：</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_6">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>200</y>
         <width>80</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>确定</string>
       </property>
      </widget>
      <widget class="QComboBox" name="cb1_2">
       <property name="geometry">
        <rect>
         <x>106</x>
         <y>100</y>
         <width>188</width>
         <height>23</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="editable">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>300</y>
         <width>131</width>
         <height>101</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
